src/modules/navigation/hooks/helpers.ts

```diff
@@ -133,9 +127,7 @@ export const computePath = (
         fromPublicFolder: isPublic
       })
     case 'shared-drive':
-      return `/shareddrive/${(file.attributes as FileAttribute).driveId}/${
-        file._id
-      }`
+      return `/shareddrive/${driveId!}/${file._id}`
```

Comment:
```suggestion
      return `/shareddrive/${driveId}/${file._id}`
```

and handle the case where driveId is undefined before or hack a fallback route

Reply:
Fixed